### ISSUE TITLE
what is Pester?

### ISSUE THREAD

**Issue by bbrendon** [pester/Pester#1825]
I'm trying PSKoans and it says to install this but nothing in the readme explains what this is.

**Comment by johlju** [comment]
Try https://pester.dev/. I can see it missing in the README.md, but it could also be added as the Website address in the repository About.

**Comment by fflaten** [comment]
I'd like to point out the description for the repo:

_Pester is the ubiquitous test and mock framework for PowerShell._

As for it's relation to PSKoans; Pester is the engine used by PSKoans. PSKoans is a basically a specialized set of Pester tests + some UI on top to make it a great learning tool for Powershell.

**Comment by bbrendon** [comment]
Ahh. In the corner. Missed that. Maybe it would make sense to hyperlink "ubiquitous test and mock framework" so someone can click on it and can understand what that means?  Someone that doesn't know much about powershell (like me) will have no idea what a "ubiquitous test and mock framework" is. Or maybe dumb it down? 

Feel free to close this if you feel it's covered. Thanks for listening.

**Comment by Andy-AO** [comment]
It is suggested that you learn software testing related knowledge, especially unit testing and test driven development(TDD). 

After learning these basic knowledge, you will understand the significance of this tool.

There is a Book about this tool called <The Pester Book> > which is much more detailed than The documentation.

**Comment by fflaten** [comment]
See current readme. The mentioned https://pester.dev/docs/quick-start link at the beginning of the current readme answers the issue. Could probably add https://pester.dev as website for repo also and close issue as fixed by #1874

**Comment by nohwnd** [comment]
Added.
